ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,203 @@
+        get {
+            return avatarState.primaryText
+        }
+
+        set {
+            avatarState.primaryText = newValue
+            title = newValue ?? ""
+        }
+    }
+
+    var secondaryText: String? {
+        get {
+            return avatarState.secondaryText
+        }
+
+        set {
+            avatarState.secondaryText = newValue
+            subtitle = newValue ?? ""
+        }
+    }
+
+    var ringColor: UIColor? {
+        get {
+            return avatarState.ringColor
+        }
+
+        set {
+            avatarState.ringColor = newValue
+        }
+    }
+
+    var backgroundColor: UIColor? {
+        get {
+            return avatarState.backgroundColor
+        }
+
+        set {
+            avatarState.backgroundColor = newValue
+        }
+    }
+
+    var foregroundColor: UIColor? {
+        get {
+            return avatarState.foregroundColor
+        }
+
+        set {
+            avatarState.foregroundColor = newValue
+        }
+    }
+
+    var presence: MSFAvatarPresence {
+        get {
+            return avatarState.presence
+        }
+
+        set {
+            avatarState.presence = newValue
+        }
+    }
+
+    var hasPointerInteraction: Bool {
+        get {
+            return avatarState.hasPointerInteraction
+        }
+
+        set {
+            avatarState.hasPointerInteraction = newValue
+        }
+    }
+
+    var isRingVisible: Bool {
+        get {
+            return avatarState.isRingVisible
+        }
+
+        set {
+            avatarState.isRingVisible = newValue
+        }
+    }
+
+    var isTransparent: Bool {
+        get {
+            return avatarState.isTransparent
+        }
+
+        set {
+            avatarState.isTransparent = newValue
+        }
+    }
+
+    var isOutOfOffice: Bool {
+        get {
+            return avatarState.isOutOfOffice
+        }
+
+        set {
+            avatarState.isOutOfOffice = newValue
+        }
+    }
+
+    private var avatarState: MSFAvatarState
+}
+
+/// View for PersonaView
+public struct PersonaView: View {
+    @ObservedObject var state: MSFPersonaViewStateImpl
+    @ObservedObject var tokens: MSFPersonaViewTokens
+
+    public init() {
+        avatar = AvatarView(style: .default, size: .xlarge)
+        state = MSFPersonaViewStateImpl(avatarState: avatar.state)
+        tokens = MSFPersonaViewTokens()
+
+        initializeState()
+    }
+
+    public var body: some View {
+        MSFListCellView(state: state, tokens: tokens, windowProvider: tokens.windowProvider)
+    }
+
+    private func initializeState() {
+        state.leadingUIView = AnyView(avatar)
+        state.leadingViewSize = .xlarge
+        state.titleTrailingAccessoryView = state.titleTrailingAccessoryView
+        state.subtitleTrailingAccessoryView = state.subtitleTrailingAccessoryView
+        state.onTapAction = state.onTapAction
+        state.layoutType = .threeLines
+    }
+
+    private let avatar: AvatarView
```

Comment:
You don't necessarily need this property ini the state class.
It's being stored in the state object property for the leading view.